index.scss

```diff
@@ -28,7 +28,7 @@
   --tabs-text: #fff;
   --tabs-border: rgba(203, 203, 203, 0.796);
 
-  --accent: #e65c00;
+  --accent: #da6200;
```

Comment:
Also slipping this change back in because I regret reverting it in a previous PR:

[`971e1f6` (#238)](https://github.com/npmgraph/npmgraph/pull/238/commits/971e1f6e3db459504e62bb8992d6a4d1312aa71c)